### PR TITLE
gluon-client-bridge: basic br-client config and wireless AP

### PR DESCRIPTION
**Pull request by tcatm**
This package provides br-client and sets up a wireless AP interface for
clients.

### CODE REVIEW

File: package/gluon-client-bridge/Makefile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+define Build/Compile
+endef
+
+define Package/gluon-client-bridge/install
+	$(CP) ./files/* $(1)/
+endef
+
+define Package/gluon-client-bridge/postinst
+#!/bin/sh
+$(call GluonCheckSite,check_site.lua)
```
Comment (NeoRaider) [inline package/gluon-client-bridge/Makefile]: Where is this check_site.lua?

File: package/gluon-client-bridge/Makefile [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include $(GLUONDIR)/include/package.mk
+
+define Package/gluon-client-bridge
+  SECTION:=gluon
+  CATEGORY:=Gluon
+  TITLE:=Provides a bridge and a wireless interface for clients to connect to
+  DEPENDS:=+gluon-core +gluon-radio-config
```
Comment (NeoRaider) [inline package/gluon-client-bridge/Makefile]: Can we somehow loosen the dependency on gluon-radio-config? Optimally, it should be possible to create a firmware containing gluon-client-bridge and maybe even gluon-batman-adv-* without gluon-radio-config if no WLAN support is needed.

Comment (tcatm) [inline package/gluon-client-bridge/Makefile]: Yes, we could do that. However, I'm not sure which package would depend on gluon-radio-config in that case. We could add it to site.mk.

**Comment by NeoRaider** [comment]
Rebased and committed to master.

**Pull request closed without merging**